src/content/docs/solvro/Strategy.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Strategia KN Solvro
+sidebar:
+  order: 1
+description: Długoterminowy plan strategiczny KN Solvro
+---
+
+import {
+  Steps,
+  Card,
+  CardGrid,
+  LinkCard,
+  Aside,
+} from "@astrojs/starlight/components";
+
+> ### <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>
+
+## Początki
+
+W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tego momentu, przez te lata nasza organizacja działała, podejmując coraz to nowsze inicjatywy. Zarządy się zmieniały, a wraz z nimi początkowa wizja zanikała.
```

Comment:
```suggestion
W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tamtego momentu, nasza organizacja podejmowała coraz to nowsze inicjatywy. Zarządy zmieniały się, a wraz z czasem początkowa wizja koła zanikała.
```

Reply:
Poprawa składni. Dodatkowo, myślę, że obarczenie upływu czasu zanikaniem wizji (koła) jest bardziej gustowne. Rozumiem, że w tym zdaniu o to chodziło, ale teraz mówimy to bezpośrednio.

Reply:
Maybe "zmieniała"?

Reply:
pravdive toto